### PR TITLE
Add blog post about next steps for lead exposure

### PR DESCRIPTION
**Pull request by justinebreuch**
## Description

Addresses: [Add blog post about next steps for lead](https://tables.area120.google.com/u/1/workspace/bxQE2CPzwu_b4sJEkotkUM/table/bX_kMNBW8LA35n6l5Hz_co/view/auSM-s1cO9l3pAsYHmOk1E)

Create a new component to render basic "blog" like pages until Wordpress is live. Will link the next-steps blog post in a follow-up PR. 

### New

Component to render basic "blog" like pages. 

## Testing and Reviewing

image

### CODE REVIEW

File: client/src/assets/blog/next_steps.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+export class NextSteps {
+  static title = 'You’re at risk for lead in your water. What now?';
+  static image = 'next-steps.jpg';
+  static content = `  <p>Learning that your area is at medium or high risk for lead may sound
```
Comment (kailamjeter) [inline client/src/assets/blog/next_steps.ts:4]: nit: is this space before the tag necessary?

File: client/src/views/BlogView.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<template>
+  <div :style='cssVars'>
+    <div class='title-section is-flex has-text-centered'>
+      <div class='h1-header-large'>
+        {{ title }}
+      </div>
+    </div>
+    <div class='section content' v-html='content'>
+    </div>
```
Comment (kailamjeter) [inline client/src/views/BlogView.vue:9]: nit: can you put this end tag on the previous line?

File: client/src/views/BlogView.vue [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  </div>
+</template>
+
+<script lang='ts'>
+import { defineComponent, PropType } from 'vue';
+
+export default defineComponent({
+  name: 'BlogView', props: {
```
Comment (kailamjeter) [inline client/src/views/BlogView.vue:17]: put props on new line

File: client/src/views/BlogView.vue [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    title: {
+      type: String, default: 'LeadOut Blog',
```
Comment (kailamjeter) [inline client/src/views/BlogView.vue:19]: same here, put default:... on a new line

File: client/src/views/BlogView.vue [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+@import '../assets/styles/global.scss';
+@import '@blueconduit/copper/scss/01_settings/design-tokens';
+@import 'bulma/sass/elements/container';
+
+.title-section {
+  @include background-image;
+
+  // Use a semi-opaque gradient to tint the background image for better text readability.
+  background-image: linear-gradient(rgba(0, 96, 100, 0.75), rgba(0, 96, 100, 0.75)), var(--background-image);
+  align-items: center;
```
Comment (kailamjeter) [inline client/src/views/BlogView.vue:45]: can you use center-container mixin?

**Review by kailamjeter — APPROVED** [review]

File: client/src/assets/blog/next_steps.ts (hunk 1 above)
Comment (justinebreuch) [reply client/src/assets/blog/next_steps.ts:4]: Removed!

File: client/src/views/BlogView.vue (hunk 2 above)
Comment (justinebreuch) [reply client/src/views/BlogView.vue:9]: Done!

File: client/src/views/BlogView.vue (hunk 3 above)
Comment (justinebreuch) [reply client/src/views/BlogView.vue:17]: Done

File: client/src/views/BlogView.vue (hunk 4 above)
Comment (justinebreuch) [reply client/src/views/BlogView.vue:19]: Done

**Pull request merged**